test/python/test_status_api.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+
+import unittest
+
+import openapi_client
+
+from . import api_config
+
+WHOAMI_FIELDS = [
+    "client_ip",
+    "user_agent",
+    "account",
+    "username",
+    "token_issued_at",
+]
+
+class TestWhoamiApi(api_config.ConfiguredTest):
+    """WhoamiApi unit test stubs"""
+
+    def setUp(self):
+        self.api = openapi_client.api.status_api.StatusApi(self.client)
+        self.bad_auth_api = openapi_client.api.status_api.StatusApi(self.bad_client)
+
+    def test_who_am_i(self):
+        """Test case for who_am_i
+
+        Provides information about the client making an API request.
```

Comment:
Let's be sure to enumerate, either in comments or print statements, exactly what we're testing for. It's best to lean towards over documenting for the sake of future contributors being able to quickly understand the goal of a test.